tsconfig.json

```diff
@@ -16,7 +16,8 @@
     "noImplicitAny": true,
     "experimentalDecorators": true,
     "typeRoots": ["./server/@types", "./node_modules/@types"],
-    "lib": ["es5", "es2015.promise"]
+    "lib": ["es5", "es2015.promise"],
+    "ignoreDeprecations": "5.0"
```

Comment:
I don't think we need to add this `ignoreDeprecations` as things flagged by that were fixed in Ben's last PR

Reply:
Ahh yes, forgot about this!

Reply:
Removed.

Reply:
👍 tbh I don't think it's needed in the template app either - but haven't had time to check yet....